index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/Styles/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+  </head>
+  <body>
+    <section class="navigation">
+      <h3>Byiringiro Moise</h3>
+      <span class="material-symbols-outlined"> menu </span>
+    </section>
+    <section class="profile">
+        <div class="introduction">
+<h1>Hey There. I'm Moise</h1>
+<h4>I'am a Software Developer</h4>
```

Comment:
- [ ] Kindly ensure that your website follows the heading hierarchy `hx (e.g.<h1>, <h2>,<h3>, etc...)`. Since you have already used the `<h1>` tag, you should process to use the `<h2>` and not the `<h4>` tag which will make your website semantic which will, in turn, increase its accessibility 👍🏽. You can check out this [link](https://www.w3.org/WAI/tutorials/page-structure/headings/) for more info